docs/getting_started/execution-modes.rst

```diff
@@ -238,6 +243,40 @@ Each task will create a new Cloud Run Job execution, giving full isolation. The
         },
     )
 
+Airflow Async
```

Comment:
Let's highlight in the title that this is experimental, similar to what we did in https://astronomer.github.io/astronomer-cosmos/configuration/render-config.html#customizing-how-nodes-are-rendered-experimental

```suggestion
Airflow Async (experimental)
```